src/systemcmds/topic_listener/CMakeLists.txt

```diff
@@ -51,6 +51,7 @@ px4_add_module(
 	SRCS
 		topic_listener.cpp
 	DEPENDS
+		platforms__common
```

Comment:
Not needed.

Reply:
inherited from previous version. this can be removed.